src/artemis/experiment_plans/fast_grid_scan_plan.py

```diff
@@ -192,7 +203,11 @@ def create_devices():
     )
 
     artemis.log.LOGGER.info("Connecting to EPICS devices...")
-    fast_grid_scan_composite.wait_for_connection()
+    try:
+        fast_grid_scan_composite.wait_for_connection()
+    except Exception as e:
+        artemis.log.LOGGER.info(e, exc_info=True)
```

Comment:
Must: As above with logging

Reply:
but now that you mention it, logging as error and raising seems good

Reply:
I'm not sure if here is the best place to do that though, I think we should be still getting those logs at the runner anyway? Logging here just makes the plan itself less readable